### ISSUE TITLE
Uncaught ReferenceError: id is not defined

### ISSUE THREAD

**Issue by Kliton** [redefiance/atom-go-find-references#14]
Uncaught ReferenceError: id is not defined error.

**Comment by jspiro** [comment]
This is still an issue.

**Comment by jspiro** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.1 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.6
**Thrown From**: [go-find-references](https://github.com/redefiance/atom-go-find-references) package 1.2.3


### Stack Trace

Uncaught ReferenceError: id is not defined

```
At /Users/jspiro/.atom/packages/go-find-references/node_modules/atom-tree-view/lib/tree-item.coffee:14

ReferenceError: id is not defined
    at /packages/go-find-references/node_modules/atom-tree-view/lib/tree-item.coffee:14:27)
    at Function.View.buildHtml (/packages/figlet/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:123:10)
    at Function.View.render (/packages/figlet/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:129:19)
    at /packages/figlet/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:627:24)
    at TreeItem.module.exports.TreeItem.initialize (/packages/go-find-references/node_modules/atom-tree-view/lib/tree-item.coffee:14:15)
    at /packages/figlet/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
    at /packages/go-find-references/node_modules/atom-tree-view/lib/tree-item.coffee:5:7)
    at /packages/go-find-references/lib/go-find-references-view.coffee:104:22
    at /packages/go-find-references/lib/go-find-references-view.coffee:101:18
    at TreeView.module.exports.TreeView.createItems (/packages/go-find-references/node_modules/atom-tree-view/lib/tree-view.coffee:60:13)
    at /packages/go-find-references/lib/go-find-references-view.coffee:100:15
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:212:1)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:543:20)
```

### Commands

```
     -1:21.3.0 command-palette:toggle (input.hidden-input)
     -1:17.6.0 core:backspace (input.hidden-input)
     -1:15.5.0 core:confirm (input.hidden-input)
     -1:15.5.0 go-find-references:toggle (input.hidden-input)
     -0:03.4.0 command-palette:toggle (input.hidden-input)
     -0:02.7.0 core:confirm (input.hidden-input)
     -0:02.7.0 go-find-references:toggle (input.hidden-input)
```

### Non-Core Packages

```
alpha-syntax 0.1.0 
an-old-hope-light-syntax 1.0.1 
ariake-dark-syntax 0.1.3 
atom-beautify 0.29.24 
atom-dark-minimal-ui 0.3.1 
atom-fix-path 0.1.0 
atom-jinja2 0.6.0 
atom-json-color 0.6.0 
atom-monokai-dark 0.4.0 
atom-singing-stars 0.3.1 
atom-ternjs 0.18.3 
atom-typescript 11.0.3 
atom-visual-studio-code-light-ui 1.1.11 
atom-visual-studio-code-ui 1.3.4 
autoclose-html 0.23.0 
autocomplete 0.47.0 
autocomplete-clang 0.11.3 
autocomplete-json 5.5.0 
autocomplete-paths 1.0.5 
autocomplete-python 1.8.63 
base16-syntax 1.8.0 
beta-syntax 0.1.1 
biome-syntax 0.2.0 
blame 0.12.0 
blessed-light-syntax 0.3.0 
block-comment 0.4.3 
blue-gray-syntax 0.4.0 
blueforest-syntax 0.3.0 
bone-a-kite 1.0.0 
boneyfied 0.1.0 
bright-dark-ui 0.5.1 
browse 1.6.1 
build-decaffeinate 0.3.1 
busy-signal 1.4.1 
carbon-blue-atom 2.1.1 
change-case 0.6.5 
circle-ci 0.10.1 
clang-format 1.29.0 
climate-syntax 5.0.0 
close-on-left 0.8.0 
close-other-panes 0.6.0 
code-school 0.6.1 
coffee-navigator 0.0.18 
cold-syntax 1.3.0 
color-picker 2.2.5 
column-overflow 0.1.0 
ctags-status 1.3.7 
dalton-syntax 1.18.0 
dark-side-of-the-moon-syntax 0.6.26 
dash 1.7.1 
declarations 1.1.3 
docblockr 0.10.3 
dracula-syntax 2.0.5 
dracula-ui 0.6.0 
dun-syntax 0.4.0 
duotone-bright-sea-syntax 2.1.0-4 
duotone-dark-amethyst-syntax 2.0.4 
duotone-dark-sea-syntax 2.1.0 
duotone-dark-space-syntax 2.1.0 
facebook-syntax 1.0.0 
figlet 0.5.4 
file-bookmark 1.0.3 
file-icon-supplement 0.8.3 
file-icons 2.1.6 
file-types 0.5.5 
filecolor 0.4.0 
find-list 0.3.0 
firecode-syntax 0.1.0 
firewatch-syntax 1.1.1 
foldername-tabs 0.2.4 
front-icons 1.14.0 
gamma-syntax 0.1.0 
git-time-machine 1.5.9 
github-atom-light-syntax 0.3.2 
glacier-dark-ui 0.5.0 
gloom 0.6.4 
glowing-cursor 0.1.2 
go-debug 1.4.0 
go-find-references 1.2.3 
go-impl 0.2.1 
go-outline 0.6.6 
go-plus 5.5.5 
go-quick-import 0.3.1 
go-signature-statusbar 1.2.0 
go-tip 2.4.0 
goextract 0.4.0 
golden-ratio 0.3.0 
gotests 0.3.0 
grammar-token-limit 0.1.1 
graphite-ui 0.15.1 
grayscale-syntax 0.1.0 
grayula-theme 1.5.0 
halflife-syntax 1.0.2 
hex 0.6.2 
highlight-bad-chars 1.0.7 
highlight-column 0.5.2 
highlight-line 0.12.0 
highlight-selected 0.13.1 
html-entities 0.4.0 
hybrid-next-syntax 0.9.2 
hyperclick 0.0.40 
immersive-nucleus-dark-ui 1.10.0 
inc 0.0.4 
indent-guide-improved 1.4.12 
indentation-jumper 0.1.2 
intentions 1.1.2 
intentions-colorpicker 1.0.1 
intentions-numbers 1.0.4 
isotope-ui 2.8.4 
js2coffee 0.19.1 
json-schema 0.1.15 
jumpy 3.1.3 
language-actionscript3 0.3.0 
language-arduino 0.4.3 
language-babel 2.62.0 
language-cmake 0.3.0 
language-diff 0.7.0 
language-docker 1.1.7 
language-fish-shell 1.0.5 
language-generic-config 1.3.0 
language-ignore 1.5.2 
language-ini 1.19.0 
language-jade 0.7.2 
language-javascript-jsx 0.3.7 
language-json5 1.1.1 
language-markdown 0.22.0 
language-nginx 0.8.0 
language-ninja 1.1.0 
language-openscad 0.4.3 
language-patch 1.2.0 
language-proto 0.8.0 
language-swift 0.5.0 
language-terraform 0.8.1 
language-todo-extra-words 0.2.0 
language-viml 1.1.3 
last-cursor-position 0.9.2 
letter-spacing 0.3.1 
linter 2.1.4 
linter-annotations 1.0.1 
linter-coffeelint 1.2.1 
linter-docker 0.2.0 
linter-jshint 3.1.2 
linter-json-lint 0.1.2 
linter-manager 0.2.12 
linter-shellcheck 1.4.4 
linter-terraform-syntax 1.2.0 
linter-ui-default 1.6.0 
linter-write-good 0.9.0 
liquid-syntax 0.3.0 
liquid-ui 0.5.0 
local-history 4.3.0 
lunar-syntax 0.1.3 
MagicPython 1.0.10 
mango-syntax 0.2.4 
markdown-writer 2.6.5 
module-syntax 0.1.5 
monocolor-amber-syntax 1.0.0 
monocolor-blue-syntax 1.0.0 
monocolor-green-syntax 1.0.0 
monocolor-light-blue-syntax 1.0.0 
monocolor-lime-syntax 1.0.0 
monocolor-orange-syntax 1.0.0 
monocolor-pink-syntax 1.0.0 
monocolor-purple-syntax 1.0.0 
monocolor-red-syntax 1.0.0 
monocolor-teal-syntax 1.0.0 
monokai-flat 1.0.1 
monokai-verbose-syntax 0.1.1 
multi-cursor 2.1.5 
multi-cursor-plus 1.2.0 
multiline-tab 0.2.0 
multirow-tabs 0.3.3 
mustang 0.1.0 
naturepurple-dark-syntax 0.1.0 
nav-panel 0.0.11 
neon-selection 0.3.2 
nightwing-syntax 0.1.0-0 
nord-atom-ui 0.11.0 
nord-atom-ui-vertical-tabs 0.0.5 
northem-dark-atom-syntax 2.1.0 
number-range 0.2.10 
number-slider 1.0.1 
numix-dark-ui 0.3.0 
one-dark-vivid-syntax 1.7.1 
open-in-developer-mode 0.3.0 
open-package 0.2.0 
orbit-syntax 0.1.1 
package-developer-toolbar 0.10.0 
path-copy 0.14.0 
peacock-syntax 1.4.0 
peacocks-in-space-syntax 0.7.0 
photon-syntax 0.9.0 
pigments 0.39.1 
plain-simple 1.1.0 
polymorph-ui 1.8.8 
prettier-standard-formatter 0.0.5 
pretty-json 1.6.3 
promote-folder-to-project 2.0.1 
protector 0.2.3 
proton 4.0.2 
proton-bat 2.0.0 
proton-monokai 0.26.0 
python-tools 0.6.9 
qolor 0.4.1 
rainy-night-syntax 0.3.2 
relative-numbers 0.8.0 
remember-file-positions 0.2.3 
satisfy-dependencies 0.3.1 
seti-monokai 0.3.1 
shell-it 0.2.1 
show-in-system 0.4.0 
show-origin 0.2.0 
simple-icons 0.1.2 
smart-tags 0.1.5 
sort-lines 0.14.0 
spacegray-dark-neue-ui 1.7.1 
spacegray-light-neue-ui 0.1.7 
spacemacs-dark-syntax 0.4.0 
spectrum-light-syntax 0.0.5 
split-diff 1.4.0 
ssh-config 0.10.1 
standard-formatter 2.8.0 
standardjs-snippets 2.6.0 
steam-pirate-syntax 0.3.0 
Stylus 3.1.0 
Sublime-Style-Column-Selection 1.7.4 
submarine 0.1.0 
summerfruit-theme 1.9.0 
swackets 0.26.0 
switch-header-source 0.22.1 
switcheroo 1.1.0 
symbols-tree-view 0.14.0 
tab-numbers 0.6.0 
tab-smart-sort 0.3.0 
tabs-closer 0.16.0 
technobabble 0.4.0 
terraform-fmt 0.3.0 
test 0.1.2 
test-gutter 0.1.1 
test-status 1.0.0 
theme-reel 1.2.0 
theme-switch 0.4.0 
todo 0.6.0 
todo-manager 0.2.10 
todo-show 1.11.0 
tool-bar 1.1.0 
tree-view-autoresize 2.0.0 
tree-view-git-status 1.4.0 
tree-view-panes 0.3.3 
tron-legacy-syntax 0.5.0 
typewriter-scroll 1.0.0 
unfancy-file-icons 0.11.0 
vim-mode-plus 0.92.1 
visual-bell 1.0.0 
zentabs 0.8.8 
```
